content/en/blog/_posts/2024-xx-xx-csi-changed-block-tracking.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog
+title: 'Kubernetes v1.33: Changed Block Tracking (CBT )API support (alpha)'
+date: 2025-XX-XX
```

Comment:
Although the schedule is going to be a bit tight, we would like this to get released on Tuesday, 13th May. The timestamp helps schedule the publication timing to be consistent with other blogs.

```suggestion
date: 2025-05-13T10:30:00-08:00
```